src/hotspot/share/runtime/sharedRuntime.cpp

```diff
@@ -3046,29 +3032,14 @@ void AdapterHandlerLibrary::lookup_simple_adapters() {
 #endif // INCLUDE_CDS
 
 address AdapterHandlerEntry::base_address() {
```

Comment:
This method is used in `print_adapter_handler_info()` only after removal of `relocate()`.
Consider using `adapter_blob->content_begin()` in  `print_adapter_handler_info()` and remove this `base_address()`